engines/bladerunner/actor.cpp

```diff
@@ -1091,10 +1098,18 @@ void Actor::speechPlay(int sentenceId, bool voiceOver) {
 		balance = CLIP<int>(balance, -127, 127);
 	}
 
+#if SUBTITLES_SUPPORT
+	_vm->_subtitles->getInGameSubsText(_id, sentenceId);
```

Comment:
this api doesnt look very future proof, for dialog between more actors api is implying will just show one sentence. maybe something like subtitles->queue(actorid, sentenceid, length)

Reply:
I agree that a queue system would be nicer and I've had it in mind as a future possible task/upgrade. But maybe it's not really needed? It seems to be working quite ok as is.
Also, I'm not sure what you mean by "future proof". Are there cases in-game where the current implementation would not work?

Reply:
it will be needed as few dialogs a bit fast, but you dont have to change right now. also this state persistence in the new code is not so nice looking. but as i wrote, you dont have to do it now